tests/event/listener_test.php

```diff
@@ -266,9 +268,10 @@ public function test_exception_errors($site, $exception)
 			'configurator'	=> $configurator,
 		]);
 
-		// Setup the listener and call the configure_media_embed method
+		// Setup the listener and call the media embed configuration methods
 		$listener = $this->get_listener();
-		$listener->configure_media_embed($event);
+		$listener->add_custom_sites($event);
+		$listener->enable_media_sites($event);
```

Comment:
Is a call to configure_url_parsing missing here?

Reply:
No. It's just not necessary in this test. Those are the methods there that will throw exceptions being tested for. configure_url_parsing will never be reached in this test..